pygit2/decl/oid.h

```diff
@@ -1,3 +1,11 @@
 typedef struct git_oid {
 	unsigned char id[20];
 } git_oid;
+
+typedef struct git_remote_head {
+        int local;
+        git_oid oid;
+        git_oid loid;
+        char *name;
+        char *symref_target;
+} git_remote_head;
```

Comment:
Please move this to `pygit2/decl/net.h` since in libgit2 this struct is defined in `net.h`, to keep things organized.

Reply:
When I do that, ffi complains: 

`
cffi.CDefError: cannot parse "int git_remote_ls(const git_remote_head ***out, size_t *size, git_remote *remote);"
<cdef source string>:554:25: before: git_remote_head
`

Which I'm guessing is related to this comment/code in pygit2/_run.py:

```
# Order matters
h_files = ...
```

What's the right solution here?

Reply:
ok, no problem then, leave it in `oid.h`, add just a comment like `this should go to net.h but ...`